src/routes/docs/products/functions/develop-locally/+page.markdoc

```diff
@@ -21,9 +21,13 @@ We use Docker to replicate the production environment for the local deployment o
 
 Use the `appwrite run functions` command to develop your function locally.
 
+| parameter       | Description                                                                                                |
```

Comment:
Also ` --user-id <user-id>` to impersonate user, which will automatically set `x-appwrite-user-id`and `x-appwrite-user-jwt` headers on context object.